exectck/src/main/java/org/apache/jdo/exectck/RunTCK.java

```diff
@@ -528,13 +693,14 @@ public void execute() throws MojoExecutionException, MojoFailureException {
               + File.separator;
       String[] metadataExtensions = {"jdo", "jdoquery", "orm", "xml", "properties"};
       String fromFileName = null;
+      File toFile = null;
       String pkgName = null;
       int startIdx = -1;
       // iterator over list of abs name of metadata files in src
       Iterator<File> fi = FileUtils.iterateFiles(new File(fromDirName), metadataExtensions, true);
       while (fi.hasNext()) {
         try {
-          fromFile = fi.next();
+          File fromFile = fi.next();
```

Comment:
I know this was already in p[lace before the refactoring. However: 

- I think this may be cleaner if the try-clause only contained the FileUtil.copy 
- Why not move() the file here?
- If the try-clause is minimized, All variables can be declared in-place: fromFileName, pkgName, toFile.

Reply:
- I agree the try clause should only contain the FileUtil.copy
- The files should be copied, because the enhanced directory should be kept in case you want to run the next tck run without enhacement.
- I agree to declare the variables in place.